sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/cryptography/cryptography_client.hpp

```diff
@@ -23,210 +23,238 @@
+     * along with all other information needed to verify it. This information should be stored
+     * with the signature.
+     */
+    Azure::Response<SignResult> SignData(
+        SignatureAlgorithm algorithm,
+        std::vector<uint8_t> const& data,
+        Azure::Core::Context const& context = Azure::Core::Context());
+
+    /**
+     * @brief Verifies the specified signature.
+     *
+     * @param algorithm The #SignatureAlgorithm to use. This must be the same algorithm used to
+     * sign the digest.
+     * @param digest The pre-hashed digest corresponding to the signature. The hash algorithm used
+     * to compute the digest must be compatable with the specified algorithm.
+     * @param signature The signature to verify.
+     * @param context A #Azure::Core::Context to cancel the operation.
+     * @return The result of the verify operation. If the signature is valid the
+     * #VerifyResult.IsValid property of the returned #VerifyResult will be set to true.
+     */
+    Azure::Response<VerifyResult> Verify(
+        SignatureAlgorithm algorithm,
+        std::vector<uint8_t> const& digest,
+        std::vector<uint8_t> const& signature,
+        Azure::Core::Context const& context = Azure::Core::Context());
+
+    /**
+     * @brief Verifies the specified signature.
+     *
+     * @param algorithm The #SignatureAlgorithm to use. This must be the same algorithm used to
+     * sign the data.
+     * @param data The data corresponding to the signature.
+     * @param signature The signature to verify.
+     * @param context A #Azure::Core::Context to cancel the operation.
+     * @return The result of the verify operation. If the signature is valid the
+     * #VerifyResult.IsValid property of the returned #VerifyResult will be set to true.
+     */
+    Azure::Response<VerifyResult> VerifyData(
+        SignatureAlgorithm algorithm,
+        Azure::Core::IO::BodyStream& data,
+        std::vector<uint8_t> const& signature,
+        Azure::Core::Context const& context = Azure::Core::Context());
+
+    /**
+     * @brief Verifies the specified signature.
+     *
+     * @param algorithm The #SignatureAlgorithm to use. This must be the same algorithm used to
+     * sign the data.
+     * @param data The data corresponding to the signature.
+     * @param signature The signature to verify.
+     * @param context A #Azure::Core::Context to cancel the operation.
+     * @return The result of the verify operation. If the signature is valid the
+     * #VerifyResult.IsValid property of the returned #VerifyResult will be set to true.
+     */
+    Azure::Response<VerifyResult> VerifyData(
+        SignatureAlgorithm algorithm,
+        std::vector<uint8_t> const& data,
+        std::vector<uint8_t> const& signature,
+        Azure::Core::Context const& context = Azure::Core::Context());
+  };
+
+}}}}} // namespace Azure::Security::KeyVault::Keys::Cryptography
```

Comment:
It's hard for me to tell by code review, are the existing APIs still in the same namespace as they were before?

Did we have any that were just in `Azure::Security::KeyVault`?